build.gradle

```diff
@@ -315,6 +315,9 @@ subprojects { subproject ->
 		}
 
 		test {
+			if (!['file-split-ftp', 'tcp-client-server-multiplex'].contains(it.project.name)) {
```

Comment:
I wouldn't do this. 
All the test must be executed in the project. 
So, no restrictions, please.
If tests in those modules are failing and then they have to be fixed.
And let's do that in the separate PRs!
If you don't like to see project failing with these exposed tests, we always can disable them for time being with some `// TODO` comment.

Reply:
Regarding
> If you don't like to see project failing with these exposed tests, we always can disable them for time being with some // TODO comment.

You're right, disabling sub-module coverage was too heavy handed, instead I've disabled the failing coverage (94d291773ecfb8c24439b0a88220c13a665c2f51).
My intention was to support continuous integration for this project, in-order to ensure users can trust that the examples are valid.
(and who knows, it might help expose upstream bugs)

Note; when I updated this branch from master (Spring Integration 6.0.2 ~ 5a5043041b37e66ba007414345e692b59628e900) the `:file-split-ftp` became green :heavy_check_mark: , only `:tcp-client-server-multiplex` still contains a single failure :stop_sign:

Reply:
@dlehammer ,

I have just fixed that TCP test: https://github.com/spring-projects/spring-integration-samples/commit/a0f710037cdde8350c0eaea97b2694534bc411ef.

Thanks for bringing this into our attention!